src/Resources/public/js/startup.js

```diff
@@ -284,7 +296,9 @@ pimcore.plugin.customermanagementframework = Class.create(pimcore.plugin.admin,
 
         this.checkNewsletterQueueStatus(statusIcon);
     },
-    postOpenDocument: function (document, type) {
+    postOpenDocument: function (event) {
+        var document = event.detail.object;
```

Comment:
```suggestion
        var document = event.detail.document;
```
https://github.com/pimcore/pimcore/blob/11.x/bundles/AdminBundle/public/js/pimcore/events.js#L71

Reply:
We should also bump `pimcore/pimcore` in composer.json to `^10.5.6` than.

Reply:
> We should also bump `pimcore/pimcore` in composer.json to `^10.5.6` than.

@aashan10 please incorporate this change as well. thanks!